# file: C:\Users\Couch\AppData\Roaming\Python\Python313\site-packages\celery\utils\serialization.py
# hypothesis_version: 6.141.1

['+00:00', '0', '1', 'Z', '__module__', '__traceback__', 'args', 'create_exception_cls', 'false', 'get_pickleable_etype', 'no', 'off', 'on', 'strtobool', 'subclass_exception', 'true', 'yes']